Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_4.h

```diff
@@ -88,7 +88,7 @@
   #endif
 #else
   #ifndef Z_STOP_PIN
-    #define Z_STOP_PIN     P1_27   // Z-STOP
+    #define Z_STOP_PIN     P0_10   // Z-STOP
```

Comment:
The pin should be defined based on where you would attach a Z endstop switch. The pin 1.27 is marked as "Z-STOP" on the board so this is correct. It happens to correspond to the Z-DIAG pin.

It makes no sense to have the `Z_STOP_PIN` be equal to the `Z_MIN_PROBE_PIN` because these need to be usable at the same time in the case of a physical Z endstop switch and a Z probe. When they share the same pin the `Z_MIN_PROBE_PIN` is not used.

I see that the intention is to allow the probe to be connected to "Z-PROBE" while still using `Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN` and getting sensorless homing. There are actually two correct approaches already available using `Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN`:
- Connect the probe to Z-STOP on the board instead of Z-PROBE, or
- Define `Z_STOP_PIN` as `P0_10` in your configuration.